charts/invenio/templates/vocabularies-configmap.yaml

```diff
@@ -0,0 +1,8 @@
+{{- if .Values.invenio.vocabularies}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: invenio-vocabularies
+data:
+{{- .Values.invenio.vocabularies | toYaml | nindent 2 }}
+{{- end }}
```

Comment:
```suggestion
{{- with .Values.invenio.vocabularies }}
apiVersion: v1
kind: ConfigMap
metadata:
  name: invenio-vocabularies
data:
  {{- toYaml . | nindent 2 }}
{{- end }}
```

Reply:
Sorry for being nit-picky. But I like reducing code repetition. It's so easy to make typos in the future if multiple places needs to be changed.

Reply:
You **have to be** nit-picky, that's what we pay you for 😜